CI/travis.osx.after_success.sh

```diff
@@ -34,19 +34,21 @@ if [[ "${MUDLET_VERSION_BUILD}" == -ptb* ]]; then
   public_test_build="true"
 fi
 
-# we deploy only certain builds
+# deploy is set to "deploy" on all Github actions. Is there a case where we would run this but not deploy?
 if [ "${DEPLOY}" = "deploy" ]; then
 
   # get commit date now before we check out an change into another git repository
   COMMIT_DATE=$(git show -s --pretty="tformat:%cI" | cut -d'T' -f1 | tr -d '-')
   YESTERDAY_DATE=$(date -v-1d '+%F' | tr -d '-')
 
-  git clone https://github.com/Mudlet/installers.git "${BUILD_DIR}/../installers"
+  # CHANGEME - setting this to my repo to test.
```

Comment:
Needed for builds to work on this PR. Will change this once https://github.com/Mudlet/installers/pull/117 is merged.